index.html

```diff
@@ -37,7 +38,7 @@
     <!--app-->
     <script src="docs/js/app.js"></script>
     <!--xeditable-->
-    <script src="dist/js/xeditable.min.js"></script>
+    <script src="dist/js/xeditable.js"></script>
```

Comment:
Sorry, I forgot to change this back when I was debugging the model binding issue that's in the bstime directive. I can change it back, it'll add another commit and hopefully you can squash it when you merge the PR, or I can leave it like it is.